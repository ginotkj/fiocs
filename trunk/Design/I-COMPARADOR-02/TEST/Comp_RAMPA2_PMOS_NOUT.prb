[DISPLAYS]
BEGIN DISPLAY LAST SESSION
ANALYSIS TRANSIENT_ANALYSIS
SYMBOL ALWAYS
TRACECOLORSCHEME NORMAL
BEGIN ANAPLOT 1
XBASE
BEGIN XAXIS
XAXISUSERNAME 0 (null)
RANGEFLAG MANUAL
TYPE LINEAR
UNIT s
BEGIN GRIDS
AUTOMATIC
MAJORNUMBERS
MAJORSTYLE LINES
MAJORPATTERN SOLID
MINORSTYLE NONE
END GRIDS
REFMIN 0
REFMAX 8.02702702702703e-009
REFUNIT s
END XAXIS
BEGIN YAXIS 1
YAXISSIDE LEFT
ACTIVE
RANGEFLAG AUTO
TYPE LINEAR
UNIT A
BEGIN GRIDS
AUTOMATIC
MAJORNUMBERS
MAJORSTYLE LINES
MAJORPATTERN SOLID
MINORSTYLE NONE
END GRIDS
BEGIN TRACE I(I_I1)+ I(I_I2)
MARKERID 0
END TRACE I(I_I1)+ I(I_I2)
END YAXIS 1
END ANAPLOT 1
BEGIN ANAPLOT 2
ACTIVE
BEGIN XAXIS
XAXISUSERNAME 0 (null)
RANGEFLAG MANUAL
TYPE LINEAR
UNIT s
BEGIN GRIDS
AUTOMATIC
MAJORNUMBERS
MAJORSTYLE LINES
MAJORPATTERN SOLID
MINORSTYLE NONE
END GRIDS
REFMIN 0
REFMAX 8.02702702702703e-009
REFUNIT s
END XAXIS
BEGIN YAXIS 1
YAXISSIDE LEFT
ACTIVE
RANGEFLAG MANUAL
TYPE LINEAR
UNIT A
BEGIN GRIDS
AUTOMATIC
MAJORNUMBERS
MAJORSTYLE LINES
MAJORPATTERN SOLID
MINORSTYLE NONE
END GRIDS
REFMIN -0.000487804878048779
REFMAX 0.00559999999999999
REFUNIT A
BEGIN TRACE ID(M_M6)
MARKERID 0
END TRACE ID(M_M6)
END YAXIS 1
END ANAPLOT 2
BEGIN ANAPLOT 3
BEGIN XAXIS
XAXISUSERNAME 0 (null)
RANGEFLAG MANUAL
TYPE LINEAR
UNIT s
BEGIN GRIDS
AUTOMATIC
MAJORNUMBERS
MAJORSTYLE LINES
MAJORPATTERN SOLID
MINORSTYLE NONE
END GRIDS
REFMIN 0
REFMAX 8.02702702702703e-009
REFUNIT s
END XAXIS
BEGIN YAXIS 1
YAXISSIDE LEFT
ACTIVE
RANGEFLAG AUTO
TYPE LINEAR
UNIT A
BEGIN GRIDS
AUTOMATIC
MAJORNUMBERS
MAJORSTYLE LINES
MAJORPATTERN SOLID
MINORSTYLE NONE
END GRIDS
BEGIN TRACE IB(M_M6)
MARKERID 0
END TRACE IB(M_M6)
END YAXIS 1
END ANAPLOT 3
BEGIN ANAPLOT 4
BEGIN XAXIS
XAXISUSERNAME 0 (null)
RANGEFLAG MANUAL
TYPE LINEAR
UNIT s
BEGIN GRIDS
AUTOMATIC
MAJORNUMBERS
MAJORSTYLE LINES
MAJORPATTERN SOLID
MINORSTYLE NONE
END GRIDS
REFMIN 0
REFMAX 8.02702702702703e-009
REFUNIT s
END XAXIS
BEGIN YAXIS 1
YAXISSIDE LEFT
ACTIVE
RANGEFLAG AUTO
TYPE LINEAR
UNIT A
BEGIN GRIDS
AUTOMATIC
MAJORNUMBERS
MAJORSTYLE LINES
MAJORPATTERN SOLID
MINORSTYLE NONE
END GRIDS
BEGIN TRACE ID(M_M7)
MARKERID 0
END TRACE ID(M_M7)
END YAXIS 1
END ANAPLOT 4
END DISPLAY LAST SESSION
