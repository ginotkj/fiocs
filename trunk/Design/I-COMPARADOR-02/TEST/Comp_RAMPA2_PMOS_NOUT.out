
**** 04/24/10 17:07:05 ******* PSpice 16.0.0 (July 2006) ****** ID# 0 ********

  


 ****     CIRCUIT DESCRIPTION


******************************************************************************



**** 04/17/10 16:10:21 ******* PSpice 16.0.0 (July 2006) ****** ID# 0 ********

 ** Profile: "TEST-transient"  [ D:\Documents\TESIS\fiocs\Design\I-COMPARADOR-02\OrCAD\i-comparador-01-pspicefiles\test\transient.si


 ****     CIRCUIT DESCRIPTION


******************************************************************************




** Creating circuit file "transient.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
.LIB "D:\Documents\TESIS\fiocs\Design\I-COMPARADOR-02\OrCAD\I-COMPARADOR-01-PSpiceFiles/i-comparador-01.lib" 
* From [PSPICE NETLIST] section of C:\OrCAD\OrCAD_16.0\tools\PSpice\PSpice.ini file:
.lib "nom.lib" 

*Analysis directives: 

.STEP V_VIN LIST 0.5 0.99 1 1.01 1.5
.TRAN  0 150n 0 10p 
.PROBE V(*) I(*) 

*.INC "..\TEST.net" 



**** INCLUDING TEST.net ****
* source I-COMPARADOR-01

M_M2         ND+ IN+ NDBIAS 0 CMOSN  
+ L=.2u  
+ W=.4u         
V_VREF         IN+ N398131 1
M_M3         ND- ND- N39211 N39211 CMOSP  
+ L=.2u  
+ W=.6u         
M_M4         ND+ ND- N39211 N39211 CMOSP  
+ L=.2u  
+ W=.6u         
M_M5         NDBIAS VBIAS 0 0 CMOSN  
+ L=.2u  
+ W=.6u         
C_C1         0 N749310  1n  
V_VQ+         N398131 0 .2
M_U2_M5         N75001 NOUT N39211 N39211 CMOSP  
+ L=.18u  
+ W=.2u         
M_U2_M6         N75001 NOUT 0 0 CMOSN  
+ L=.18u  
+ W=.18u         
M_M6         NOUT ND+ N39211 N39211 CMOSP  
+ L=.2u  
+ W=.8u         
V_VBIAS         VBIAS 0 .33
M_M7         NOUT VBIAS 0 0 CMOSN  
+ L=.2u  
+ W=.4u         
V_VQ-         N43113 0 .2
V_VIN         IN- N43113 1.5
R_R1         N75001 N749310  500k  
M_M1         ND- IN- NDBIAS 0 CMOSN  
+ L=.2u  
+ W=.4u         
V_VDD         N39211 0 3.3

I_I1         0 NOUT DC 0Adc AC 0Aac
+PULSE 0 4m 2n 30p 150p 0p 100n
I_I2         0 NOUT DC 0Adc AC 0Aac
+PULSE 0 2m 2n 60p 2n 50p 100n

.END

Mosfet M_M2, model CMOSN: Using BSIM VERSION 3.1 or lower
WARNING -- Mosfet M_M2, model CMOSN: Pd = 0 is less than W
WARNING -- Mosfet M_M2, model CMOSN: Ps = 0 is less than W
Mosfet M_M5, model CMOSN: Using BSIM VERSION 3.1 or lower
WARNING -- Mosfet M_M5, model CMOSN: Pd = 0 is less than W
WARNING -- Mosfet M_M5, model CMOSN: Ps = 0 is less than W
Mosfet M_U2_M6, model CMOSN: Using BSIM VERSION 3.1 or lower
WARNING -- Mosfet M_U2_M6, model CMOSN: Pd = 0 is less than W
WARNING -- Mosfet M_U2_M6, model CMOSN: Ps = 0 is less than W
Mosfet M_M3, model CMOSP: Using BSIM VERSION 3.1 or lower
WARNING -- Mosfet M_M3, model CMOSP: Pd = 0 is less than W
WARNING -- Mosfet M_M3, model CMOSP: Ps = 0 is less than W
Mosfet M_U2_M5, model CMOSP: Using BSIM VERSION 3.1 or lower
WARNING -- Mosfet M_U2_M5, model CMOSP: Pd = 0 is less than W
WARNING -- Mosfet M_U2_M5, model CMOSP: Ps = 0 is less than W
Mosfet M_M6, model CMOSP: Using BSIM VERSION 3.1 or lower
WARNING -- Mosfet M_M6, model CMOSP: Pd = 0 is less than W
WARNING -- Mosfet M_M6, model CMOSP: Ps = 0 is less than W
**** 04/24/10 17:07:05 ******* PSpice 16.0.0 (July 2006) ****** ID# 0 ********

  


 ****     MOSFET MODEL PARAMETERS


******************************************************************************




               CMOSN           CMOSP           
               NMOS            PMOS            
  T_Measured   27              27            
   T_Current   27              27            
       LEVEL    7               7            
           L  100.000000E-06  100.000000E-06 
           W  100.000000E-06  100.000000E-06 
         VTO     .299302        -.401632     
          KP  470.883200E-06  470.883200E-06 
       GAMMA    0               0            
      LAMBDA    0               0            
         RSH    6               6.4          
          IS    1.000000E-15    1.000000E-15 
          JS  100.000000E-06  100.000000E-06 
          PB     .980128         .821464     
        PBSW     .8              .8008       
          CJ  815.376400E-06    1.165977E-03 
        CJSW  166.520200E-12  122.005600E-12 
          MJ     .606194         .425655     
        MJSW     .169221         .1001       
        CGSO  500.000000E-12  500.000000E-12 
        CGDO  500.000000E-12  500.000000E-12 
        CGBO    1.000000E-12    1.000000E-12 
         TOX    4.400000E-09    4.400000E-09 
          XJ  100.000000E-09  100.000000E-09 
       UCRIT   10.000000E+03   10.000000E+03 
       DELTA     .01             .01         
      DIOMOD    2               2            
          K1     .542117         .598774     
          K2    -.035133     -630.087300E-09 
        LETA    0               0            
        WETA    0               0            
          U0  268.8192        120.6045       
       XPART     .5              .5          
        VTH0     .299302        -.401632     
          K3    1.000000E-03     .081248     
          W0  100.000000E-09    1.000000E-06 
         NLX  228.107500E-09   42.193340E-09 
        DVT0     .882304         .948505     
        DVT1     .324768         .682461     
          UA   -1.551592E-09    1.640708E-09 
          UB    2.710709E-18    1.272651E-21 
          UC   46.128640E-12 -100.000000E-12 
        VSAT  116.006100E+03  182.237700E+03 
        RDSW  150             726.9944       
        VOFF    -.091862        -.15         
     NFACTOR    1.807002         .921842     
        PCLM    2.027608         .151904     
      PDIBL1     .784214         .056431     
      PDIBL2    -.042689         .012341     
       DROUT    1                .470272     
      PSCBE1    2.314588E+09    4.022317E+09 
      PSCBE2  500.000000E-12    7.751373E-09 
          A0    1.525088         .904176     
          A1    6.962179E-03    8.069322E-03 
          A2     .524367         .560796     
       NPEAK  235.490000E+15  415.890000E+15 
         LDD    0               0            
        LITL   36.331810E-09   36.331810E-09 
         UC1  -56.000000E-12  -56.000000E-12 
        PVAG                     .124556     
        KETA    -.015366         .020751     
        ETA0    5.535422E-03    1.016564E-03 
        ETAB -671.692500E-06   -1.910199E-03 
         K3B    5.015682       19.7189       
        DVT2    -.056267        -.3          
        DSUB    3.692569E-03     .014544     
         AGS     .400988         .131884     
       DVT1W    0               0            
       DVT2W    0               0            
        PRWG     .340675        -.049475     
        PRWB    -.183865        -.420982     
     PDIBLCB    -.1            -1.000000E-03 
         DWG    5.852475E-09  -30.588980E-09 
         DWB   11.572970E-09  -11.142310E-09 
          B0  -44.157810E-09    1.909663E-06 
          B1                    5.000000E-06 
        LINT   20.701890E-09   26.472300E-09 
        WINT   12.149120E-09                 
         DLC   20.701890E-09   26.472300E-09 
         DWC   12.149120E-09    0            
          CF    0               0            
        NOIA  100.000000E+18    9.900001E+18 
        NOIB   50.000000E+03    2.400000E+03 
        NOIC   -1.400000E-12    1.400000E-12 
       LKETA   -1.853360E-03   -8.835292E-03 
       WKETA    3.404068E-03     .030613     
       PVSAT    1.045662E+03   50            
       PKETA    -.0116         -5.087778E-03 
       PVTH0   -3.570294E-03  633.898700E-06 
         PUA  -25.147800E-12  364.727200E-12 
         PUB   21.902820E-24                 
         PU0   -2.447231        7.159787     
       PRDSW   -1.670209       -5            
       PETA0 -100.000000E-06  100.000000E-06 
         PK2    3.193075E-03   31.405140E-06 
         VTM     .025864         .025864     
       PBSWG     .8              .8008       
       MJSWG     .169221         .1001       
       CJSWG  330.000000E-12  422.000000E-12 


Mosfet M_M2, model CMOSN: Using BSIM VERSION 3.1 or lower
WARNING -- Mosfet M_M2, model CMOSN: Pd = 0 is less than W
WARNING -- Mosfet M_M2, model CMOSN: Ps = 0 is less than W
Mosfet M_M5, model CMOSN: Using BSIM VERSION 3.1 or lower
WARNING -- Mosfet M_M5, model CMOSN: Pd = 0 is less than W
WARNING -- Mosfet M_M5, model CMOSN: Ps = 0 is less than W
Mosfet M_U2_M6, model CMOSN: Using BSIM VERSION 3.1 or lower
WARNING -- Mosfet M_U2_M6, model CMOSN: Pd = 0 is less than W
WARNING -- Mosfet M_U2_M6, model CMOSN: Ps = 0 is less than W
Mosfet M_M3, model CMOSP: Using BSIM VERSION 3.1 or lower
WARNING -- Mosfet M_M3, model CMOSP: Pd = 0 is less than W
WARNING -- Mosfet M_M3, model CMOSP: Ps = 0 is less than W
Mosfet M_U2_M5, model CMOSP: Using BSIM VERSION 3.1 or lower
WARNING -- Mosfet M_U2_M5, model CMOSP: Pd = 0 is less than W
WARNING -- Mosfet M_U2_M5, model CMOSP: Ps = 0 is less than W
Mosfet M_M6, model CMOSP: Using BSIM VERSION 3.1 or lower
WARNING -- Mosfet M_M6, model CMOSP: Pd = 0 is less than W
WARNING -- Mosfet M_M6, model CMOSP: Ps = 0 is less than W
Mosfet M_M2, model CMOSN: Using BSIM VERSION 3.1 or lower
WARNING -- Mosfet M_M2, model CMOSN: Pd = 0 is less than W
WARNING -- Mosfet M_M2, model CMOSN: Ps = 0 is less than W
Mosfet M_M5, model CMOSN: Using BSIM VERSION 3.1 or lower
WARNING -- Mosfet M_M5, model CMOSN: Pd = 0 is less than W
WARNING -- Mosfet M_M5, model CMOSN: Ps = 0 is less than W
Mosfet M_U2_M6, model CMOSN: Using BSIM VERSION 3.1 or lower
WARNING -- Mosfet M_U2_M6, model CMOSN: Pd = 0 is less than W
WARNING -- Mosfet M_U2_M6, model CMOSN: Ps = 0 is less than W
Mosfet M_M3, model CMOSP: Using BSIM VERSION 3.1 or lower
WARNING -- Mosfet M_M3, model CMOSP: Pd = 0 is less than W
WARNING -- Mosfet M_M3, model CMOSP: Ps = 0 is less than W
Mosfet M_U2_M5, model CMOSP: Using BSIM VERSION 3.1 or lower
WARNING -- Mosfet M_U2_M5, model CMOSP: Pd = 0 is less than W
WARNING -- Mosfet M_U2_M5, model CMOSP: Ps = 0 is less than W
Mosfet M_M6, model CMOSP: Using BSIM VERSION 3.1 or lower
WARNING -- Mosfet M_M6, model CMOSP: Pd = 0 is less than W
WARNING -- Mosfet M_M6, model CMOSP: Ps = 0 is less than W
**** 04/24/10 17:07:05 ******* PSpice 16.0.0 (July 2006) ****** ID# 0 ********

  


 ****     INITIAL TRANSIENT SOLUTION       TEMPERATURE =   27.000 DEG C

 ****     CURRENT STEP                     V_VIN =     .5   


******************************************************************************



 NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE


(  IN+)    1.2000  (  IN-)     .7000  (  ND+)     .4010  (  ND-)    3.0081      

( NOUT)    3.2988  (VBIAS)     .3300  (N39211)    3.3000 (N43113)     .2000     

(N75001) 110.3E-09 (NDBIAS)     .4006 (N398131)     .2000                       

(N749310) 110.3E-09                   




    VOLTAGE SOURCE CURRENTS
    NAME         CURRENT

    V_VREF       0.000E+00
    V_VQ+        0.000E+00
    V_VBIAS      0.000E+00
    V_VQ-        0.000E+00
    V_VIN        0.000E+00
    V_VDD       -4.986E-07

    TOTAL POWER DISSIPATION   1.65E-06  WATTS


Mosfet M_M2, model CMOSN: Using BSIM VERSION 3.1 or lower
WARNING -- Mosfet M_M2, model CMOSN: Pd = 0 is less than W
WARNING -- Mosfet M_M2, model CMOSN: Ps = 0 is less than W
Mosfet M_M5, model CMOSN: Using BSIM VERSION 3.1 or lower
WARNING -- Mosfet M_M5, model CMOSN: Pd = 0 is less than W
WARNING -- Mosfet M_M5, model CMOSN: Ps = 0 is less than W
Mosfet M_U2_M6, model CMOSN: Using BSIM VERSION 3.1 or lower
WARNING -- Mosfet M_U2_M6, model CMOSN: Pd = 0 is less than W
WARNING -- Mosfet M_U2_M6, model CMOSN: Ps = 0 is less than W
Mosfet M_M3, model CMOSP: Using BSIM VERSION 3.1 or lower
WARNING -- Mosfet M_M3, model CMOSP: Pd = 0 is less than W
WARNING -- Mosfet M_M3, model CMOSP: Ps = 0 is less than W
Mosfet M_U2_M5, model CMOSP: Using BSIM VERSION 3.1 or lower
WARNING -- Mosfet M_U2_M5, model CMOSP: Pd = 0 is less than W
WARNING -- Mosfet M_U2_M5, model CMOSP: Ps = 0 is less than W
Mosfet M_M6, model CMOSP: Using BSIM VERSION 3.1 or lower
WARNING -- Mosfet M_M6, model CMOSP: Pd = 0 is less than W
WARNING -- Mosfet M_M6, model CMOSP: Ps = 0 is less than W
**** 04/24/10 17:07:05 ******* PSpice 16.0.0 (July 2006) ****** ID# 0 ********

  


 ****     INITIAL TRANSIENT SOLUTION       TEMPERATURE =   27.000 DEG C

 ****     CURRENT STEP                     V_VIN =     .99  


******************************************************************************



 NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE


(  IN+)    1.2000  (  IN-)    1.1900  (  ND+)    2.7206  (  ND-)    2.9686      

( NOUT)    3.2908  (VBIAS)     .3300  (N39211)    3.3000 (N43113)     .2000     

(N75001) 138.2E-09 (NDBIAS)     .7872 (N398131)     .2000                       

(N749310) 138.2E-09                   




    VOLTAGE SOURCE CURRENTS
    NAME         CURRENT

    V_VREF       0.000E+00
    V_VQ+        0.000E+00
    V_VBIAS      0.000E+00
    V_VQ-        0.000E+00
    V_VIN        0.000E+00
    V_VDD       -5.261E-07

    TOTAL POWER DISSIPATION   1.74E-06  WATTS


Mosfet M_M2, model CMOSN: Using BSIM VERSION 3.1 or lower
WARNING -- Mosfet M_M2, model CMOSN: Pd = 0 is less than W
WARNING -- Mosfet M_M2, model CMOSN: Ps = 0 is less than W
Mosfet M_M5, model CMOSN: Using BSIM VERSION 3.1 or lower
WARNING -- Mosfet M_M5, model CMOSN: Pd = 0 is less than W
WARNING -- Mosfet M_M5, model CMOSN: Ps = 0 is less than W
Mosfet M_U2_M6, model CMOSN: Using BSIM VERSION 3.1 or lower
WARNING -- Mosfet M_U2_M6, model CMOSN: Pd = 0 is less than W
WARNING -- Mosfet M_U2_M6, model CMOSN: Ps = 0 is less than W
Mosfet M_M3, model CMOSP: Using BSIM VERSION 3.1 or lower
WARNING -- Mosfet M_M3, model CMOSP: Pd = 0 is less than W
WARNING -- Mosfet M_M3, model CMOSP: Ps = 0 is less than W
Mosfet M_U2_M5, model CMOSP: Using BSIM VERSION 3.1 or lower
WARNING -- Mosfet M_U2_M5, model CMOSP: Pd = 0 is less than W
WARNING -- Mosfet M_U2_M5, model CMOSP: Ps = 0 is less than W
Mosfet M_M6, model CMOSP: Using BSIM VERSION 3.1 or lower
WARNING -- Mosfet M_M6, model CMOSP: Pd = 0 is less than W
WARNING -- Mosfet M_M6, model CMOSP: Ps = 0 is less than W
**** 04/24/10 17:07:05 ******* PSpice 16.0.0 (July 2006) ****** ID# 0 ********

  


 ****     INITIAL TRANSIENT SOLUTION       TEMPERATURE =   27.000 DEG C

 ****     CURRENT STEP                     V_VIN =    1     


******************************************************************************



 NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE


(  IN+)    1.2000  (  IN-)    1.2000  (  ND+)    2.9655  (  ND-)    2.9655      

( NOUT)    1.1958  (VBIAS)     .3300  (N39211)    3.3000 (N43113)     .2000     

(N75001)    1.4639 (NDBIAS)     .7935 (N398131)     .2000                       

(N749310)    1.4639                   




    VOLTAGE SOURCE CURRENTS
    NAME         CURRENT

    V_VREF       0.000E+00
    V_VQ+        0.000E+00
    V_VBIAS      0.000E+00
    V_VQ-        0.000E+00
    V_VIN        0.000E+00
    V_VDD       -7.152E-05

    TOTAL POWER DISSIPATION   2.36E-04  WATTS


Mosfet M_M2, model CMOSN: Using BSIM VERSION 3.1 or lower
WARNING -- Mosfet M_M2, model CMOSN: Pd = 0 is less than W
WARNING -- Mosfet M_M2, model CMOSN: Ps = 0 is less than W
Mosfet M_M5, model CMOSN: Using BSIM VERSION 3.1 or lower
WARNING -- Mosfet M_M5, model CMOSN: Pd = 0 is less than W
WARNING -- Mosfet M_M5, model CMOSN: Ps = 0 is less than W
Mosfet M_U2_M6, model CMOSN: Using BSIM VERSION 3.1 or lower
WARNING -- Mosfet M_U2_M6, model CMOSN: Pd = 0 is less than W
WARNING -- Mosfet M_U2_M6, model CMOSN: Ps = 0 is less than W
Mosfet M_M3, model CMOSP: Using BSIM VERSION 3.1 or lower
WARNING -- Mosfet M_M3, model CMOSP: Pd = 0 is less than W
WARNING -- Mosfet M_M3, model CMOSP: Ps = 0 is less than W
Mosfet M_U2_M5, model CMOSP: Using BSIM VERSION 3.1 or lower
WARNING -- Mosfet M_U2_M5, model CMOSP: Pd = 0 is less than W
WARNING -- Mosfet M_U2_M5, model CMOSP: Ps = 0 is less than W
Mosfet M_M6, model CMOSP: Using BSIM VERSION 3.1 or lower
WARNING -- Mosfet M_M6, model CMOSP: Pd = 0 is less than W
WARNING -- Mosfet M_M6, model CMOSP: Ps = 0 is less than W
**** 04/24/10 17:07:05 ******* PSpice 16.0.0 (July 2006) ****** ID# 0 ********

  


 ****     INITIAL TRANSIENT SOLUTION       TEMPERATURE =   27.000 DEG C

 ****     CURRENT STEP                     V_VIN =    1.01  


******************************************************************************



 NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE


(  IN+)    1.2000  (  IN-)    1.2100  (  ND+)    3.1929  (  ND-)    2.9623      

( NOUT) 279.2E-06  (VBIAS)     .3300  (N39211)    3.3000 (N43113)     .2000     

(N75001)    3.3000 (NDBIAS)     .7998 (N398131)     .2000                       

(N749310)    3.3000                   




    VOLTAGE SOURCE CURRENTS
    NAME         CURRENT

    V_VREF       0.000E+00
    V_VQ+        0.000E+00
    V_VBIAS      0.000E+00
    V_VQ-        0.000E+00
    V_VIN        0.000E+00
    V_VDD       -1.149E-07

    TOTAL POWER DISSIPATION   3.79E-07  WATTS


Mosfet M_M2, model CMOSN: Using BSIM VERSION 3.1 or lower
WARNING -- Mosfet M_M2, model CMOSN: Pd = 0 is less than W
WARNING -- Mosfet M_M2, model CMOSN: Ps = 0 is less than W
Mosfet M_M5, model CMOSN: Using BSIM VERSION 3.1 or lower
WARNING -- Mosfet M_M5, model CMOSN: Pd = 0 is less than W
WARNING -- Mosfet M_M5, model CMOSN: Ps = 0 is less than W
Mosfet M_U2_M6, model CMOSN: Using BSIM VERSION 3.1 or lower
WARNING -- Mosfet M_U2_M6, model CMOSN: Pd = 0 is less than W
WARNING -- Mosfet M_U2_M6, model CMOSN: Ps = 0 is less than W
Mosfet M_M3, model CMOSP: Using BSIM VERSION 3.1 or lower
WARNING -- Mosfet M_M3, model CMOSP: Pd = 0 is less than W
WARNING -- Mosfet M_M3, model CMOSP: Ps = 0 is less than W
Mosfet M_U2_M5, model CMOSP: Using BSIM VERSION 3.1 or lower
WARNING -- Mosfet M_U2_M5, model CMOSP: Pd = 0 is less than W
WARNING -- Mosfet M_U2_M5, model CMOSP: Ps = 0 is less than W
Mosfet M_M6, model CMOSP: Using BSIM VERSION 3.1 or lower
WARNING -- Mosfet M_M6, model CMOSP: Pd = 0 is less than W
WARNING -- Mosfet M_M6, model CMOSP: Ps = 0 is less than W
**** 04/24/10 17:07:05 ******* PSpice 16.0.0 (July 2006) ****** ID# 0 ********

  


 ****     INITIAL TRANSIENT SOLUTION       TEMPERATURE =   27.000 DEG C

 ****     CURRENT STEP                     V_VIN =    1.5   


******************************************************************************



 NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE


(  IN+)    1.2000  (  IN-)    1.7000  (  ND+)    3.3000  (  ND-)    2.9205      

( NOUT) 11.95E-06  (VBIAS)     .3300  (N39211)    3.3000 (N43113)     .2000     

(N75001)    3.3000 (NDBIAS)    1.1928 (N398131)     .2000                       

(N749310)    3.3000                   




    VOLTAGE SOURCE CURRENTS
    NAME         CURRENT

    V_VREF       0.000E+00
    V_VQ+        0.000E+00
    V_VBIAS      0.000E+00
    V_VQ-        0.000E+00
    V_VIN        0.000E+00
    V_VDD       -1.483E-07

    TOTAL POWER DISSIPATION   4.89E-07  WATTS



          JOB CONCLUDED

**** 04/24/10 17:07:05 ******* PSpice 16.0.0 (July 2006) ****** ID# 0 ********

  


 ****     JOB STATISTICS SUMMARY

 ****     CURRENT STEP                     V_VIN =    1.5   


******************************************************************************



  Total job time (using Solver 1)   =        9.10
